Sources/Logging/Logging.swift

```diff
@@ -786,7 +912,7 @@ extension Logger {
     /// - parameters:
     ///     - label: An identifier for the creator of a `Logger`.
     public init(label: String) {
-        self.init(label: label, LoggingSystem.factory(label))
+        self.init(label: label, LoggingSystem.factory(label, .bootstrapped))
```

Comment:
could you elaborate with this does? the Logger defining ".bootstrapped" state on the factory seems odd?

Reply:
We can remove the "bootstrapped" computed property but the logic needs to be here: get the metadata provider instance the system was bootstrapped with when we create a new logger.

I'll make this 

```
self.init(label: label, LoggingSystem.factory(label, LoggingSystem.metadataProvider))
```

if you think that's nicer

Reply:
I did that then